en/docs/guides/identity-verification/idvp-ui.md

```diff
@@ -0,0 +1,307 @@
+| required        | Whether the input field is required or not.                                                                                                                                 | true                             |
+| defaultValue    | Default value for the input field.                                                                                                                                          | “default value”                  |
+| displayOrder    | Defines the display order the configuration properties. If the attribute is not defined, elements are ordered according to the order they are defined in the metadata.json. | 1                                |
+| dataComponentId | The data component id of the input field which can be used in test automation.                                                                                              | “idvp-config-settings-token”     |
+
+
+Common Attributes - Text and Numeric Input Field Types
+Apart from the above-mentioned attributes, the following attributes are supported by every text and numeric input field type.
+
+| Attribute            | Description                                                     | Example                        |
+|----------------------|-----------------------------------------------------------------|--------------------------------|
+| Placeholder          | A placeholder value for the input field.                        | “Enter the token from Onfido.” |
+| validationRegex      | A regex pattern to be validated against the input field value.  | /^https?:\/\//                 |
+| regexValidationError | An error message to display if the regex validation failed.     | “The URL is not secure”        | 
+| maxLength            | Maximum number of characters that can be included in the field. | 1024                           |
+| minLength            | minimum number of characters that can be included in the field. | 1                              |
+
+
+Specific Attributes of Input Field Types
+Number
+
+| Attribute  | Description                                            | Example |
+|------------|--------------------------------------------------------|---------|
+| minValue   | The minimum value that can be used in the input field. | 1       |
+| maxValue   | The maximum value that can be used in the input field. | 100     |
+
+Dropdown
+
+| Attribute  | Description                                                                                       | Example |
+|------------|---------------------------------------------------------------------------------------------------|---------|
+| Options    | An array of options to be used in the drop-down. Each option has a value and an associated label. |```[{"value":"nic", "label":"NIC"}]```|
+
+
+
+Important:
+
+The “defaultValue” attribute of the dropdown type should also be in the above format.
+
+Example
```

Comment:
As this example is long, shall we add it in an expandable?

Reply:
Lets add all examples in expandables. Doing that will make the doc more readable.

Reply:
Resolved